tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -276,27 +294,361 @@ test.describe( 'Set up accounts', () => {
+				} );
+
+				test( 'select MC Account 2 and click "Connect" button should see Merchant Center "Connected"', async () => {
+					await Promise.all( [
+						// Mock Merchant Center create accounts
+						setUpAccountsPage.mockMCCreateAccountWebsiteNotClaimed(),
+
+						// Mock Merchant Center as connected with ID 12345
+						setUpAccountsPage.mockMCConnected( 23456 ),
+					] );
+
+					// Select MC Account 2 from the options
+					const mcAccountsSelect =
+						setUpAccountsPage.getMCAccountsSelect();
+					await mcAccountsSelect.selectOption( {
+						label: 'MC Account 2 ・ https://example.com (23456)',
+					} );
+
+					// Click connect button
+					const connectButton = setUpAccountsPage.getConnectButton();
+					await connectButton.click();
+					await page.waitForLoadState(
+						LOAD_STATE.DOM_CONTENT_LOADED
+					);
+
+					const mcConnectedLabel =
+						setUpAccountsPage.getGoogleComboConnectedLabel();
+					await expect( mcConnectedLabel ).toContainText(
+						'Connected'
+					);
+
+					const mcDescriptionRow =
+						setUpAccountsPage.getMCDescriptionRow();
+					await expect( mcDescriptionRow ).toContainText(
+						`Merchant Center ID: 23456`
+					);
+				} );
+			} );
+
+			test.describe( 'click "Or, create a new Merchant Center account"', () => {
+				test.beforeAll( async () => {
+					await Promise.all( [
+						// Mock Jetpack as connected.
+						setUpAccountsPage.mockJetpackConnected(),
+
+						// Mock google as connected.
+						setUpAccountsPage.mockGoogleConnected(
+							'google@example.com'
+						),
+
+						// Mock Google Ads as connected.
+						setUpAccountsPage.mockAdsAccountConnected(),
+						setUpAccountsPage.mockAdsStatusClaimed(),
+
+						// Mock merchant center as not connected.
+						setUpAccountsPage.mockMCNotConnected(),
+
+						// Mock merchant center has accounts
+						setUpAccountsPage.mockMCHasAccounts(),
+					] );
+
+					await setUpAccountsPage.goto();
+				} );
+
+				test( 'should see see a modal to ensure the intention of creating a new account', async () => {
+					// Click 'Or, create a new Merchant Center account'
+					const mcFooterButton =
+						setUpAccountsPage.getMCCardFooterButton();
+					await mcFooterButton.click();
+					await page.waitForLoadState(
+						LOAD_STATE.DOM_CONTENT_LOADED
+					);
+
+					const modalHeader = setUpAccountsPage.getModalHeader();
+					await expect( modalHeader ).toContainText(
+						'Create Google Merchant Center Account'
+					);
+
+					const modalCheckbox = setUpAccountsPage.getModalCheckbox();
+					await expect( modalCheckbox ).toBeEnabled();
+
+					const modalPrimaryButton =
+						setUpAccountsPage.getModalPrimaryButton();
+					await expect( modalPrimaryButton ).toContainText(
+						'Create account'
+					);
+					await expect( modalPrimaryButton ).toBeDisabled();
+
+					// Select the checkbox, the button should be enabled.
+					await modalCheckbox.click();
+					await expect( modalPrimaryButton ).toBeEnabled();
+				} );
```

Comment:
Looks like the same test is already in place.

https://github.com/woocommerce/google-listings-and-ads/blob/8fba3e4ecd1d8b12f365706545d47f66976a1770/tests/e2e/specs/setup-mc/step-1-accounts.test.js#L342-L364

Reply:
Test removed @eason9487